main.css

```diff
@@ -0,0 +1,172 @@
+html,body{
+  /*default doc styling*/
+  position: relative;
+  height: 100%;
+  background: #fff;
+}
+
+body{
+  /*default body styling*/
+  margin-top: 0 !important;#
+  padding-top: 0 !important;
+  margin: 0 auto;
+  display: block;
+  width: 1400px;
+  background-image: url("./image/your_verse_hero.jpg");
+  background-attachment: scroll;
+  background-repeat: no-repeat;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  color: #333;
+}
+
+
+.header{
+  /*header size*/
+  width: 100%;
+}
+
+table {
+  /* table stylesheet*/
+  width: 60%;
+  margin-left: 150px;
+  padding-top: 20px;
+  border-color: transparent;
+  height: 35px;
+  border: 0px solid transparent;
+  border-spacing: 0px;
+}
+
+table tr{
+  /*table row styling*/
+  border: 0px solid transparent;
+  border-collapse: collapse;
+  background-image: linear-gradient(white, #111, white);
+}
+
+table tr td{
+  /*table data styling*/
+  border: 0px solid transparent;
+  border-collapse: collapse;
+  background-color: #333;
+  opacity: 0.7;
+  height: 35px;
+}
+
+table tr td:hover{
+  /*hovering behaviour for navbar*/
+  color: #333;
+  opacity: 1;
+}
+
+button{
+  /*navbar buttons default setting*/
+  border: 2px solid #333;
+  background-color: #333;
+  opacity: 0.7;
+  height: 35px;
+  width: 110px;
+  color: white;
+}
+
+input{
+  /*search box styling*/
+  background-image: url("./image/search.png");
+  background-repeat: no-repeat; background-position: 5px center;
+  position: static; right: 172px;top: 30px;
+  height: 18px; width: 100px;
+  background-color: inherit;
+  border-radius: 10px;
+  border-color: #222; color: #fff;
+}
+
+.main{
+  /*main section positioning*/
+  text-align: center;
+  height: 100%; width: 100%;
```

Comment:
I think this is causing the page to render incorrectly. At least it's what happened in my browser (Firefox)